src/components/CloudHeader/_cloud-header.scss

```diff
@@ -0,0 +1,119 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/helper-mixins';
+
+$cloud-header-link-color: $inverse-01 !default;
+$cloud-header-icon-color: $inverse-01 !default;
+$cloud-header-menu-color: $brand-02 !default;
+$cloud-header-background-color: $color__blue-90 !default;
+
+.cloud-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: $cloud-header-background-color;
+  color: $cloud-header-link-color;
+  height: rem(48px);
+  width: 100%;
+
+  a {
```

Comment:
Should we update this from an element selector to a class selector to help de-couple the markup from the styling?

Reply:
Changed